raiden/smart_contracts/ChannelManagerContract.sol

```diff
@@ -114,6 +120,24 @@ contract ChannelManagerContract {
     }
 
     function newChannel(address partner, uint settle_timeout) returns (address channel) {
+        address channel_address;
```

Comment:
Okay. This is not what I meant. I wanted you to make a function that `deleteChannel` that you call from this function. The logic should still be in the library. It will require a bit of refactoring and not just moving code from one place to another, but it should be possible.